contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -182,15 +187,21 @@ contract VaultHub is PausableUntilWithRoles {
         VaultHubStorage storage $ = _getVaultHubStorage();
         uint256 limit = _offset + _limit > $.sockets.length - 1 ? $.sockets.length - 1 - _offset : _limit;
         VaultInfo[] memory batch = new VaultInfo[](limit);
+        uint256 startIndex = _offset + 1;
```

Comment:
can declare it inside return :thought_balloon: (might save bytecode a bit probably)